pangeo_forge_recipes/config.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass, field
+from typing import Optional, Union
+
+from pangeo_forge_recipes.storage import CacheFSSpecTarget, FSSpecTarget
+from pangeo_forge_recipes.transforms import RequiredAtRuntimeDefault
+
+
+@dataclass
+class RecipeConfig:
+    """a simple class that SHOULD house all injection spec values
+    from `pangeo_forge_recipes.injections.py:get_injection_specs()`
+
+    This way folks can use it to pass injection spec values around
+    on to custom transforms or partials in their recipe:
+
+    ```python
+    from config import RecipeConfig
+
+    @dataclass
+    class MyCustomTransform(beam.PTransform):
+        # note that `MyCustomTransform` is not part of the
+        # pangeo_forge_recipes.injections.py:get_injection_specs()
+        # but now it can be used anyway in practice
+        target_root: None
+
+    recipe_config = RecipeConfig()
+
+    beam.Create() | MyCustomTransform(target_root=recipe_config.target_root)
```

Comment:
```suggestion
    from config import Config

    @dataclass
    class MyCustomTransform(beam.PTransform):
        # note that `MyCustomTransform` is not part of the
        # pangeo_forge_recipes.injections.py:get_injection_specs()
        # but now it can be used anyway in practice
        target_root: None

    config = Config()

    beam.Create() | MyCustomTransform(target_root=config.target_root)
```